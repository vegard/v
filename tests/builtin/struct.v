@t := struct {
	foo: u64;
	bar: u64;
	baz: struct {
		x: u64;
	};
};

x := t();
x.foo = u64 10;
x.bar = u64 23;
x.baz.x = u64 999;
print x.foo;
print x.bar;
print x.baz.x;
