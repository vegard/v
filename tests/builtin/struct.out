10
23
999
